lib/warder/arguments.rb

```diff
@@ -0,0 +1,104 @@
+require 'optparse'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.